src/legacy/core_plugins/timelion/public/components/timelion_expression_input_helpers.ts

```diff
@@ -0,0 +1,268 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { get, startsWith } from 'lodash';
+import PEG from 'pegjs';
+import * as monacoEditor from 'monaco-editor/esm/vs/editor/editor.api';
+
+// @ts-ignore
+import grammar from 'raw-loader!../chain.peg';
+
+import { i18n } from '@kbn/i18n';
+import { ITimelionFunction, TimelionFunctionArgs } from '../../common/types';
+import { ArgValueSuggestions, FunctionArg, Location } from '../services/arg_value_suggestions';
+
+const Parser = PEG.generate(grammar);
```

Comment:
@timroes  @lukeelmers  
Have questions about `PEG`. I see that we have the ability to generate code not at runtime.
e.g.  using `Kibana/tasks/config/peg.js` task. In this case code will be added into bundle?

Which method is more preferable?